dist/css/main.css

```diff
@@ -15697,24 +15697,37 @@ body.theme-footer-white .footer-site hr {
   margin-bottom: 2.4rem;
 }
 
+@media (min-width: 992px) {
+  .footer-govt-logo a {
+    margin: 0;
+  }
+}
+
 .footer-nav-links a {
-  color: white;
+  color: white !important;
```

Comment:
Why do we need to use `!important` here?

Reply:
![screen shot 2018-12-11 at 3 36 14 pm](https://user-images.githubusercontent.com/26725710/49774720-8a102d00-fd5a-11e8-9af8-ad5fcb665c94.png)

The `color: white;` property is being overridden by the `theme-picker` from the user. The `!important` is a quick fix, but I'm open to better solutions :)

Reply:
Done!